src/main/java/com/saucelabs/bamboo/sod/config/SODKeys.java

```diff
@@ -79,5 +79,5 @@ private SODKeys() {}
     public static final String SAUCE_CONNECT_V3 = CUSTOM_PREFIX + "sauceConnect3";
 
     public static final String SAUCE_SESSION_ID = "sauceondemand.sessionId";
-    public static final String TEMP_TUNNEL_ID = "TEMP_TUNNEL_ID";
+    public static final String TUNNEL_ID = "SAUCE_TUNNEL_IDENTIFIER";
```

Comment:
it has weird logic which checks if `SAUCE_`  is prefixed for environment variables